dist/tools/shellcheck/check.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2018 Freie Universität Berlin
+#                    Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+SHELLCHECK_CMD="$(which shellcheck)"
+export SHELLCHECK_OPTS="-e SC1090"
+
+if tput colors &> /dev/null && [ "$(tput colors)" -ge 8 ]; then
+    CERROR=$'\e[1;31m'
+    CRESET=$'\e[0m'
+else
+    CERROR=
+    CRESET=
+fi
+
+: "${RIOTTOOLS:=${PWD}/dist/tools}"
+. "${RIOTTOOLS}"/ci/changed_files.sh
+
+FILES=$(FILEREGEX='(?=*.sh$)' changed_files)
```

Comment:
this regex doesn't work with macOS, and also is no proper/standard? regex anyway. Using `FILES=$(FILEREGEX='(.*\.sh$)' changed_files)` it works, allowing any file name which ends on `.sh`.

Reply:
Thanks for testing this, I've replaced it with your suggestion.